src/cli/env.js

```diff
@@ -1,5 +1,11 @@
+import { env } from 'process';
+
 const parseEnv = () => {
-    // Write your code here 
+    // Write your code here
+    console.log(Object.entries(env)
+        .filter(([key, value]) => key.startsWith('RSS_'))
+        .map(([key, value]) => `${key}=${value}`)
+        .join('; '));
```

Comment:
For more readability you can move calculation from console.log to a variable

```suggestion
    const value = Object.entries(env)
        .filter(([key, value]) => key.startsWith('RSS_'))
        .map(([key, value]) => `${key}=${value}`)
        .join('; ');
    console.log(value);
```